features/product/managing_products/select_product_taxons_for_product.feature

```diff
@@ -0,0 +1,23 @@
+@managing_products
+Feature: Modify existing product taxon
+    In order to specify in which taxon a product is available
+    As an Administrator
+    I want to be able to change taxon of a product
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store classifies its products as "Clothes" and "T-Shirts"
+        And the store has a "T-Shirt" configurable product
+        And the product "T-shirt" belongs to taxon "Clothes"
+        And I am logged in as an administrator
+
+    @api
+    Scenario: Modifying existing product taxon
+        When I change that the "T-Shirt" product belongs to the "T-Shirts" taxon
```

Comment:
If I see correctly, there is different behaviour in the API than in UI in the same step. We should probably only add a new taxon to the product in this step, not remove the existing one.